datafusion-physical-expr/src/tdigest/mod.rs

```diff
@@ -708,29 +714,12 @@ mod tests {
 
     #[test]
     fn test_int64_uniform() {
-        let values = (1i64..=1000).map(|v| ScalarValue::Int64(Some(v)));
-
-        let t = TDigest::new(100);
-        let t = t.merge_unsorted(values).unwrap();
-
-        assert_error_bounds!(t, quantile = 0.1, want = 100.0);
-        assert_error_bounds!(t, quantile = 0.5, want = 500.0);
-        assert_error_bounds!(t, quantile = 0.9, want = 900.0);
-        assert_state_roundtrip!(t);
-    }
-
-    #[test]
-    fn test_int64_uniform_with_nulls() {
```

Comment:
remove this test case because now `merge_unsorted_f64` will not accept None, the None filter should be test in `try_as_f64`

https://github.com/apache/arrow-datafusion/blob/136e9e9803b17c0a248939e904e22f218f3efb4b/datafusion-physical-expr/src/tdigest/mod.rs#L53-L60